vnext/Shared/DevSettings.h

```diff
@@ -24,7 +24,11 @@ namespace react {
 enum class JSIEngineOverride : int32_t {
   Default = 0, // No JSI, will use the legacy ExecutorFactory
   Chakra = 1, // Use the JSIExecutorFactory with ChakraRuntime
+
+#if 0 // Deprecated
```

Comment:
Why not just removing it?

Reply:
Historical purposes.
Devs could ask themselves why there is no value `2`.